app/ethermint_test.go

```diff
@@ -1,24 +1,25 @@
 package app
 
 import (
+	"encoding/json"
 	"os"
 	"testing"
 
 	"github.com/stretchr/testify/require"
 
+	"github.com/cosmos/cosmos-sdk/simapp"
+
 	abci "github.com/tendermint/tendermint/abci/types"
 	"github.com/tendermint/tendermint/libs/log"
 	dbm "github.com/tendermint/tm-db"
-
-	"github.com/cosmos/cosmos-sdk/codec"
 )
 
 func TestEthermintAppExport(t *testing.T) {
 	db := dbm.NewMemDB()
-	app := NewEthermintApp(log.NewTMLogger(log.NewSyncWriter(os.Stdout)), db, nil, true, map[int64]bool{}, 0)
+	app := NewEthermintApp(log.NewTMLogger(log.NewSyncWriter(os.Stdout)), db, nil, true, map[int64]bool{}, DefaultNodeHome, 0, MakeEncodingConfig(), simapp.EmptyAppOptions{})
 
-	genesisState := ModuleBasics.DefaultGenesis()
-	stateBytes, err := codec.MarshalJSONIndent(app.cdc, genesisState)
+	genesisState := NewDefaultGenesisState()
+	stateBytes, err := json.MarshalIndent(genesisState, "", "  ")
```

Comment:
why `"  "` ?

Reply:
i think this is just to format the output by adding an indent